package.json

```diff
@@ -61,19 +61,19 @@
 	"dependencies": {
 		"@liskhq/lisk-api-client": "5.1.6",
 		"@liskhq/lisk-chain": "0.3.4",
```

Comment:
Don't we need to update this to beta?

@liskhq/lisk-api-client@6.0.0-beta.0
@liskhq/lisk-chain@0.4.0-beta.0

Reply:
This is correct. We need the older versions to connect to Lisk Core v3+.